plugin/externalPluginEntry.go

```diff
@@ -58,29 +65,60 @@ func mungeToMethods(input []interface{}) (map[string]interface{}, error) {
 	return methods, nil
 }
 
-func (e decodedExternalPluginEntry) toExternalPluginEntry() (*externalPluginEntry, error) {
-	if e.Name == "" {
+func (e decodedExternalPluginEntry) toExternalPluginEntry(schemaKnown bool, isRoot bool) (*externalPluginEntry, error) {
+	if len(e.Name) <= 0 {
```

Comment:
Why change this? The comparison to empty string is considered ideomatic.

Reply:
Apparently this is a bit more idiomatic -- I saw this type of comparison being used in the Go source. If I'm wrong, happy to change it back.

Reply:
I guess they both are: https://stackoverflow.com/a/18595217/2048059. Since it's consistent with the other tests here, the new code is fine.